src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1000,6 +1000,11 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                     if (@('NativeCommandErrorMessage','NativeCommandError') -notcontains $_.FullyQualifiedErrorId -and @('CategoryView','ConciseView','DetailedView') -notcontains $ErrorView)
                                     {
                                         $myinv = $_.InvocationInfo
+                                        $errorColor = ''
+                                        if ($Host.UI.SupportsVirtualTerminal) {
+                                            $errorColor = $PSStyle.Formatting.Error
+                                        }
```

Comment:
Maybe change to the following?
```
$retValue = if ($myinv -and $myinv.MyCommand)
{
  ....
}
elseif (....)
{
  ....
}

if ($retValue -and $Host.UI.SupportsVirtualTerminal) {
    $PSStyle.Formatting.Error + $retValue  ### do you want to add the `reset` sequence here?
}
```

Reply:
`reset` isn't needed as this is just the first part of the error message